doc/admin/install/aws.md

```diff
@@ -32,10 +32,14 @@ If you're just starting out, we recommend [installing code-server locally](../..
   ```
   >example: `ssh -i "/Users/John/Downloads/TestInstance.pem" ubuntu@ec2-3-45-678-910.compute-1.amazonaws.co`
 - You should see a prompt for your EC2 instance like so<img src="../../assets/aws_ubuntu.png">
-- At this point it is time to download the `code-server` binary. We will of course want the linux version. Make sure you copy the link for the latest linux version on our [releases page](https://github.com/codercom/code-server/releases)
-- With the URL in the clipboard, run:
+- At this point it is time to download the `code-server` binary. We will of course want the linux version.
+- Find and download the latest Linux release from this URL:
   ```
-  wget https://github.com/codercom/code-server/releases/download/0.1.4/code-server-linux
+  https://github.com/codercom/code-server/releases/latest
+  ```
+- Extract the downloaded tar.gz file with this command, for example:
+  ```
+  tar -xvzf code-server-1.31.1-100-linux-x64.tar.gz
```

Comment:
Would it make sense to leave the `wget` command? We could say something like:

1. Find the latest version number on the [releases page](https://github.com/codercom/code-server/releases/latest).
2. Replace `{version}` in the following command with the version found on the releases page and run it (or just copy the download URL from the releases page):
```
wget https://github.com/codercom/code-server/releases/download/{version}/code-server-{version}-linux-x64.tar.gz
```
This would also let us use `{version}` as a placeholder in all the other commands.

Reply:
I've made it like this now.